x-pack/filebeat/input/httpjson/value_tpl.go

```diff
@@ -40,6 +41,11 @@ const (
 var (
 	errEmptyTemplateResult = errors.New("the template result is empty")
 	errExecutingTemplate   = errors.New("the template execution failed")
+	errMinWrongNoOfArgs    = errors.New("template: :1:2: executing \"\" at <min>: wrong number of args for min: want at least 1 got 0")
+	errMaxWrongNoOfArgs    = errors.New("template: :1:2: executing \"\" at <max>: wrong number of args for max: want at least 1 got 0")
+	errMinUnknownType      = errors.New("template: :1:2: executing \"\" at <min 1 \"b\" -2>: error calling min: unknown type for \"b\" (string)")
+	errMaxUnknownType      = errors.New("template: :1:2: executing \"\" at <max 1 \"b\" -2>: error calling max: unknown type for \"b\" (string)")
```

Comment:
Use `` ` `` for these to reduce `\`.

I think the error messages could be clearer. I don't see what the ":1:2" and "b" refer to. Prefer just format a message at the error location with the type and the value. Oh! These are for tests. Please put them as literals in the test table.

Reply:
I put them here in this fashion cause the other tests in this test suit seemed to the same, so in order to maintain consistency with the others, chose this route.

Reply:
These 4 errors look like they only exist for the purpose of error string matching in test cases. I think keeping them local to the test case they are associated would be better for readability of the test cases. But at a minimum they should be moved to a `*_test.go` file.